sys-kernel/coreos-kernel/coreos-kernel-4.2.2-r10.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2014 CoreOS, Inc.
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+COREOS_SOURCE_REVISION="-r2"
+inherit coreos-kernel
+
+DESCRIPTION="CoreOS Linux kernel"
+KEYWORDS="amd64 arm64"
+
+DEPEND="
+	app-arch/gzip
+	app-shells/bash
+	sys-apps/coreutils
+	sys-apps/findutils
+	sys-apps/grep
+	sys-apps/ignition
+	sys-apps/less
+	sys-apps/sed
+	sys-apps/shadow
+	sys-apps/systemd
+	sys-apps/util-linux
+	sys-fs/btrfs-progs
+	sys-fs/mdadm
+	sys-kernel/dracut
+	virtual/udev
+	"
+RDEPEND="${DEPEND}"
```

Comment:
This is going to pull ignition into the root filesystem of prod images. Just use DEPEND instead.